draft-selander-lake-authz.md

```diff
@@ -398,62 +465,48 @@ If U does not recognize the EAD item or the EAD item contains information that U
 
 If all verifications are passed, then U sends EDHOC message_3.
 
-The Signature_or_MAC_3 field calculated using the private key corresponding to PK_U is either a signature or a MAC depending on EDHOC method.
-
-EAD_3 MAY contain a certificate enrollment request, see e.g., CSR specified in {{I-D.ietf-cose-cbor-encoded-cert}}, or other request which the device is now authorized to make.
-
 EDHOC message_3 may be combined with an OSCORE request, see {{I-D.ietf-core-oscore-edhoc}}.
 
 #### Processing in V
 
-V performs the normal EDHOC verifications of message_3. V may retrieve CRED_I from W, after V learnt ID_CRED_I from U.
+V performs the normal EDHOC verifications of message_3. V may retrieve CRED_U from a Credential Database, after having learnt ID_CRED_I from U.
 
 ## Authenticator <-> Authorization Server (V <-> W) {#V-W}
 
-V and W are assumed to be able to authenticate and set up a secure connection, out of scope for this specification, for example TLS 1.3 authenticated with certificates. V is assumed to authenticate with the public key PK_V, see {{domain-auth}}.
-
-This secure connection protects the Voucher Request/Response Protocol (see protocol between V and W in {{fig-protocol}}).
+It is assumed that V and W  have set up a secure connection, W has accessed the authentication credential CRED_V to be used in the EDHOC session between V and with U, and that W has verified that V is in possession of the private key corresponding to CRED_V, see {{domain-auth}}.
 
-The hash of EDHOC message_1, H(message_1), acts as session identifier of the Voucher Request/Response protocol, and binds together instances of the two protocols (U<->V and V<->W).
+V and W run the Voucher Request/Response protocol over the secure connection. The hash of EDHOC message_1, H(message_1), acts as session identifier and binds together instances of the two protocols (U<->V and V<->W).
 
 ### Voucher Request {#voucher_request}
 
 #### Processing in V
 
-Unless already in place, V and W establish a secure connection. V uses H(message_1) as a session identifier associated to this connection with W. If the same value of H(message_1) is already used for a connection with this or other W, the protocol SHALL be discontinued.
+V uses H(message_1) as a session identifier associated to this connection with W. If the same value of H(message_1) is already used for a connection with this or other W, the protocol SHALL be discontinued.
 
-V sends the voucher request to W. The Voucher Request SHALL be a CBOR array as defined below:
+V sends the voucher request to W. The Voucher Request SHALL be the CBOR array \[false, message_1\] where "false" is the CBOR simple value with encoding 0xf4.
```

Comment:
Correct. I wanted to avoid mistaking this message_1 from a message_1 intended for W, let's discuss in the meeting.

Reply:
I think if it arrives at the `/voucherrequest` endpoint AND it is wrapped in a `bstr`, it would not be confused by a regular `message_1` intended for W. But yes let's discuss in the meeting.

Reply:
I think that it will have a unique Content-Type: when it is POSTed.

Reply:
@malishav Will you look at content-types when you update Section 5?

Reply:
Missed to respond to this before. Yes, I will take a look at that.